api/api.go

```diff
@@ -54,18 +54,6 @@ func (s *Server) Serve() (err error) {
 	go s.ServeRPC()
 	go s.ServeWebapp()
 
-	peerBootstrapped := func(peerId string) {
```

Comment:
Note for others: I asked @ramfox to remove this. It's a carry-over from older libp2p days that we no longer need. The contents of `node.StartOnlineServices` have been incorporated into the new `GoOnline` call